app/src/main/java/com/codepath/apps/restclienttemplate/RestClient.java

```diff
@@ -26,7 +27,7 @@ public class RestClient extends OAuthBaseClient {
 	public static final BaseApi REST_API_INSTANCE = FlickrApi.instance(FlickrApi.FlickrPerm.WRITE); // Change this
 	public static final String REST_URL = "twitter.comp";
```

Comment:
```suggestion
	public static final String REST_URL = "twitter.com";
```
I think you have a typo.